internal/zinc-compile-core/src/main/scala/sbt/internal/inc/javac/DiagnosticsReporter.scala

```diff
@@ -241,11 +242,20 @@ object DiagnosticsReporter {
           case _ => noPositionInfo
         }
 
+      val pointerSpace = pointer.map[String] { p =>
+        lineContent.toList.take(p.intValue()).map {
+          case '\t' => '\t'
+          case _    => ' '
+        }.mkString
+      }
```

Comment:
this doesn't work?
```suggestion
        lineContent.take(p.intValue()).map {
          case '\t' => '\t'
          case _    => ' '
        }
      }
```

Reply:
I followed the code from: https://github.com/sbt/zinc/blob/4eacff2a9bf5c8750bfc5096955065ce67f4e68a/internal/compiler-bridge/src/main/scala/xsbt/DelegatingReporter.scala#L137-L140.

Should we address it in both places?